### DISCUSSION TITLE
'tls handshake failure' - Problem to access nextcloud aio interface

### DISCUSSION THREAD

**Discussion by Peterfilho** [nextcloud/all-in-one#3962] [Questions] [answered]
I installed nextcloud AIO from docker and set my domain to nextcloud tunel over domain (https://nextcloud.lidermaravalhas.acttecnologia.com.br)

with in the logs It's returns this error:
```
{"level":"error","ts":1703251138.7804308,"logger":"tls.obtain","msg":"will retry","error":"[nextcloud.lidermaravalhas.acttecnologia.com.br] Obtain: [nextcloud.lidermaravalhas.acttecnologia.com.br] solving challenge: nextcloud.lidermaravalhas.acttecnologia.com.br: [nextcloud.lidermaravalhas.acttecnologia.com.br] authorization failed: HTTP 400 urn:ietf:params:acme:error:tls - 2606:4700:3036::6815:286b: remote error: tls: handshake failure (ca=https://acme-staging-v02.api.letsencrypt.org/directory)","attempt":7,"retrying_in":600,"elapsed":1812.599546005,"max_duration":2592000}
```

In cloudflare i already set this path but no one works

`HTTP://localhost:11000`
`HTTPS://localhost:8443`

If i access with local IP on port 8080 its all fine:

![image](https://github.com/nextcloud/all-in-one/assets/13772779/7032db4f-41fe-4605-965c-86f418782601)

I was thinking if need some apache configuration on vm or something like.

What can i do?

**Comment by szaimen** [comment, marked as answer]
Maybe this helps? https://github.com/nextcloud/all-in-one/discussions/2105

**Comment by Peterfilho** [comment]
I've already seen this before but not help, so i tryed to review NAT rules and was it. After this, i not recomend to disable the 'SKIP_DOMAIN_VALIDATION' because it can be used as debug. So my problem is that i was passing from 8443 to 11000 but not working, so i change firewall 443 default port and used 443 to nextcloud, so NAT was 443 external to > 443 internal. After this change it works fine.
